### ISSUE TITLE
Option to add custom options for fenced blocks

### ISSUE THREAD

**Issue by georgtree** [apnadkarni/ruff#84]
Hello!
Is it possible to add custom options to \``` fences?
For example, I want to see syntax highlighting of tcl in markdown, it needs \```tcl format, but ruff removes any words near \```.
Thank you in advance!

**Comment by apnadkarni** [comment]
Not for the upcoming release as I just want to get 9.0 support out. But for the future, how is the ``` to be rendered for the various output formats?

**Comment by apnadkarni** [comment]
Would it be acceptable to add an option to fences like `-align` and `-caption` that are already present. Something like

``````
``` -caption "Source listing 1" -language tcl
some Tcl code
```
``````

Just immediately following the fence with a language identifier has some ambiguity with the "transform" feature for diagrams.

**Comment by georgtree** [comment]
Hello, thank you, I will try it out!

**Comment by apnadkarni** [comment]
Just to clarify - that `-language` option is not currently implemented.

**Comment by apnadkarni** [comment]
Implemented for 2.4.2 so you can say 

`````
```tcl
set x "Tcl code"
```
`````
